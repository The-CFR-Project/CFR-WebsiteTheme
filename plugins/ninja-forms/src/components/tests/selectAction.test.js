import { render, cleanup } from '@testing-library/react';
import { form } from './mockData';
import { SelectAction } from '../';

describe( 'submissions-select-action', () => {
	afterEach( cleanup );
	it( 'Matches snapshot', () => {
		const setEmailAction = jest.fn();

		const { container } = render(
			<SelectAction form={ form } setAction={ setEmailAction } />
		);
		expect( container ).toMatchSnapshot();
	} );
} );
