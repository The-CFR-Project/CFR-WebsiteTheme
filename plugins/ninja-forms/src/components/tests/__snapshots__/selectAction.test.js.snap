// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`submissions-select-action Matches snapshot 1`] = `
<div>
  <div
    class="components-custom-select-control"
  >
    <label
      class="components-custom-select-control__label"
      for="downshift-1-toggle-button"
      id="downshift-1-label"
    >
      Select Email Action:
    </label>
    <button
      aria-expanded="false"
      aria-haspopup="listbox"
      aria-label="Select Email Action:"
      class="components-button components-custom-select-control__button is-small"
      id="downshift-1-toggle-button"
      type="button"
    >
      <svg
        aria-hidden="true"
        class="components-custom-select-control__button-icon"
        focusable="false"
        height="24"
        role="img"
        viewBox="0 0 24 24"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M17.5 11.6L12 16l-5.5-4.4.9-1.2L12 14l4.5-3.6 1 1.2z"
        />
      </svg>
    </button>
    <ul
      aria-hidden="true"
      aria-labelledby="downshift-1-label"
      class="components-custom-select-control__menu"
      id="downshift-1-menu"
      role="listbox"
      tabindex="-1"
    />
  </div>
</div>
`;
